leader-election-impl/src/main/java/com/palantir/leader/LeadershipEvents.java

```diff
@@ -54,22 +54,23 @@ public LeadershipEvents(MetricRegistry metrics) {
     }
 
     public void proposedLeadershipFor(long round) {
-        leaderLog.info("Proposing leadership", SafeArg.of("round", round));
+        leaderLog.info("Proposing leadership for {}", SafeArg.of("round", round));
         proposedLeadership.mark();
     }
 
     public void gainedLeadershipFor(PaxosValue value) {
-        leaderLog.info("Gained leadership", SafeArg.of("value", value));
+        leaderLog.info("Gained leadership for {}", SafeArg.of("value", value));
         gainedLeadership.mark();
     }
 
     public void lostLeadershipFor(PaxosValue value) {
-        leaderLog.info("Lost leadership", SafeArg.of("value", value));
+        leaderLog.info("Lost leadership for {}", SafeArg.of("value", value));
         lostLeadership.mark();
     }
 
     public void noQuorum(PaxosValue value) {
-        leaderLog.warn("The most recent known information says this server is the leader, but there is no quorum right now",
+        leaderLog.warn("The most recent known information says this server is the leader,"
+                        + " but there is no quorum right now {}",
```

Comment:
The statement doesnt make sense if the parameter is logged at the end. Lets have, `The most recent known information says this server is the leader, but there is no quorum right now. The paxos value is {}`